style/sass/README.md

```diff
@@ -27,29 +27,178 @@
 * Use a `%` unit for the amount/weight when using Sass's color functions: `darken($color, 20%)`, not `darken($color, 20)`
 * Use a trailing comma after each item in a map, including the last item.
 
-## Order
-
-* Use alphabetical order for declarations.
-* Place `@extends` and `@includes` at the top of your declaration list.
-* Place media queries directly after the declaration list.
-* Place pseudo-classes and pseudo-elements second.
-* Place nested elements third.
-* Place nested classes fourth.
-
 ## Selectors
 
-* Don't use ID's for style.
-* Avoid over-qualified selectors: `h1.page-title`, `div > .page-title`
 * Use meaningful names: `$visual-grid-color` not `$color` or `$vslgrd-clr`.
-* Be consistent about naming conventions for classes. For instance, if a project is using BEM, continue using it, and if it's not, do not introduce it.
 * Use ID and class names that are as short as possible but as long as necessary.
 * Avoid nesting more than 3 selectors deep.
 * Avoid using comma delimited selectors.
```

Comment:
Curious about this one. Is this a "if you're using comma delimited selector list you should should probably write a mixin" kind of thing? Or is it pushing for encapsulating style within a component, and not trying to be too "efficient" with a vanilla CSS-style authoring approach.

I'm thinking situations like `:hover`/`:focus`, where you would want to combine.